conf/st2.conf.sample

```diff
@@ -100,6 +100,16 @@ logging = conf/logging.exporter.conf
 # Directory to dump data to.
 dump_dir = /opt/stackstorm/exports/
 
+[garbagecollector]
```

Comment:
It's ok to have **a component** per package... However it requires to ship all virtualenv and co with the package, this might be okay... But doesn't seem to be so good when there are many packages. Each package - additional same bundle of same everything, which is ~15-20mb of packaged size)) So less packages is better then having many packages, imo.